### PR TITLE
Updated help docs

### PR DESCRIPTION
**Pull request by pavangudiwada**

### CODE REVIEW

File: docs/help.rst [hunk 1]
```diff
@@ -247,3 +247,29 @@ Alert Manager is not working
...
                               requests:
                                 storage: 10Gi
+
+
+CRD issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. details:: CustomResourceDefinition.apiextensions.k8s.io "prometheuses.monitoring.coreos.com" is invalid: metadata.annotations: Too long
+
+      This is often a CRD issue which can be fixed with :ref:`these instructions <Manual Upgrade>` To verify this, open the ArgoCD error logs and look for more details.
```
Comment (aantn) [inline docs/help.rst:257]: @pavangudiwada this is the wrong fix. You don't fix by applying a manual upgrade, you fix by checking off the "server-side apply" option

Comment (aantn) [reply docs/help.rst:257]: (The image below is right, the text above is not.)

File: docs/setup-robusta/gitops/argocd.rst [hunk 2]
```diff
@@ -120,8 +120,9 @@ Finally, run ``robusta logs`` from your cli and make sure there is no error.
 
     On some Robusta versions, the sync might fail with ``CustomResourceDefinition.apiextensions.k8s.io “prometheuses.monitoring.coreos.com” is invalid: metadata.annotations: Too long: must have at most 262144 bytes``.
 
-    To solve it, use the workaround proposed `here <https://github.com/prometheus-community/helm-charts/issues/1500#issuecomment-1132907207>`_
+    To solve it, use the workaround proposed :ref:`here <CRD Issues>`.
```
Comment (aantn) [inline docs/setup-robusta/gitops/argocd.rst:123]: Wrong fix. This is the server-side apply one.

Comment (aantn) [reply docs/setup-robusta/gitops/argocd.rst:123]: https://blog.ediri.io/kube-prometheus-stack-and-argocd-25-server-side-apply-to-the-rescue

File: docs/help.rst (hunk 1 above)
Comment (aantn) [reply docs/help.rst:257]: https://blog.ediri.io/kube-prometheus-stack-and-argocd-25-server-side-apply-to-the-rescue

**Review by aantn — CHANGES_REQUESTED** [review]

File: docs/help.rst (hunk 1 above)
Comment (pavangudiwada) [reply docs/help.rst]: Fixed it

File: docs/setup-robusta/gitops/argocd.rst (hunk 2 above)
Comment (pavangudiwada) [reply docs/setup-robusta/gitops/argocd.rst:123]: Fixed at the reference

File: docs/help.rst [hunk 3]
```diff
@@ -247,3 +247,29 @@ Alert Manager is not working
...
                               requests:
                                 storage: 10Gi
+
+
+CRD issues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. details:: CustomResourceDefinition.apiextensions.k8s.io "prometheuses.monitoring.coreos.com" is invalid: metadata.annotations: Too long
+
+      This is often a CRD issue which can be fixed by enabling service-side apply option as shown below. Check out `this blog <https://blog.ediri.io/kube-prometheus-stack-and-argocd-25-server-side-apply-to-the-rescue>`_ to learn more.
```
Comment (aantn) [inline docs/help.rst:257]: Server-side not service-side

File: docs/setup-robusta/gitops/argocd.rst (hunk 2 above)
Comment (aantn) [reply docs/setup-robusta/gitops/argocd.rst:123]: Can you link it directly from here to save the user another click?

**Review by aantn — DISMISSED** [review]
2 minor comments. Please fix then merge

**Review by aantn — APPROVED** [review]

**Pull request merged**